src/app/auth/components/registration/registration.component.scss

```diff
@@ -0,0 +1,44 @@
+::ng-deep app-registration-page {
+  margin: 45px auto 0 auto;
+  display: flex;
+  justify-content: center;
+  background: #f2f2f2;
+  max-width: 330px;
+  padding: 15px;
+}
+
+.regestration-form {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+
+  h2 {
+    margin: 0;
```

Comment:
It is better to use "& h2" form in scss. Also I would recommend to use BEM naming system for classes in template. It will make scss part more easy and nice